### ISSUE TITLE
Search input should have a "Search button"

### ISSUE THREAD

**Issue by zufrieden** [swiss/styleguide#200]
![screenshot 2014-08-08 15 50 22](https://cloud.githubusercontent.com/assets/451236/3857398/f7e7f712-1f02-11e4-8239-ea55720e8ba7.png)
